src/views/layouts/base.njk

```diff
@@ -60,6 +60,7 @@
     <script src="{{ assetPath }}/javascripts/removeFilter.js" type="text/javascript"></script>
     <script src="{{ assetPath }}/javascripts/backLink.js" type="text/javascript"></script>
     <script src="{{ assetPath }}/javascripts/searchBar.js" type="text/javascript"></script>
+    <script src="{{ assetPath }}/javascripts/linkStyling.js" type="text/javascript"></script>
```

Comment:
Does this need to be included on everypage? Probably a wider question with all our js. We should look to only include eventlisteners on pages that require it.

Reply:
I've moved the scripts to the appropriate files